moabb/datasets/gigadb.py

```diff
@@ -85,38 +93,40 @@ def _get_single_subject_data(self, subject):
                         'F6', 'F8', 'FT8', 'FC6', 'FC4', 'FC2', 'FCz', 'Cz',
                         'C2', 'C4', 'C6', 'T8', 'TP8', 'CP6', 'CP4', 'CP2',
                         'P2', 'P4', 'P6', 'P8', 'P10', 'PO8', 'PO4', 'O2']
+        # fmt: on
```

Comment:
Another one

Reply:
notes on `Weibo2014.py` apply here (and further) too

Reply:
I changed those ones (double quotes + black-ish style)